book/website/ethical-research/activism/activism-cultural-change.md

```diff
@@ -0,0 +1,141 @@
+(er-activism-cultural-change)=
+
+# Cultural Change
+
+(er-activism-cultural-change-what)=
+## What is cultural change?
+
+Culture is the social environment in which you partake. 
+Culture has an influence on your views, values, concerns and your identity. 
+Culture can be difficult to change as it is hard to address directly and shared values can be deeply ingrained ({cite:ps}`Kotter2012change`). 
+Instead, norms of behaviour may be easier to change. 
+
+Here, cultural change is defined as “inspiring a change in behaviour in persons/organisations, with a lasting impact, strengthening the core values of persons/organisation"
+
+This section will continue to describe where you will encounter cultural change. 
+This is followed by short summaries of works on cultural change, written by Adrienne Marie Brown, William and Susan Bridges and John P. Kotter. 
+
+(er-activism-cultural-change-when)=
+## Where/when will you encounter cultural change?
+As your social environment is continuously changing, you're likely to experience some degree of cultural change on a daily basis. 
+This might not feel like it, because the actual change that takes place will take months or even years. 
+
+Generally, you will either be affected by cultural change, or, you want to invoke cultural change by adjusting existing practices yourself. 
+
+When you're affected by cultural change taking place in your environment, it may be difficult to change your practises and behaviour. 
```

Comment:
```suggestion
When you are affected by cultural change taking place in your environment, it may be difficult to change your practises and behaviour. 
```